tensorflow/python/data/experimental/service/server_lib.py

```diff
@@ -200,6 +203,15 @@ def join(self):
     """
     self._server.join()
 
+  def stop(self):
```

Comment:
Thanks. Do you intend to add it to both the dispatcher and worker? It seems only added to the dispatcher, and the tests are failing with 

```
AttributeError: 'WorkerServer' object has no attribute 'stop'
```

Reply:
Added `stop` to `WorkerServer`.